### PR TITLE
Fix missing file from sdist

### LINKED ISSUE #166: Tests can't be executed from sdist tarball
**Describe the bug**
Some test files are missing from the sdist tarball, e.g. `tests/golden-metadata/big/blob-with-errors.jwt`.

**To Reproduce**
Build the package from the sdist tarball and execute the tests: https://hydra.nixos.org/build/215307928/nixlog/2

**Expected behavior**
Tests pass.

**Environment**
 - OS name + version: NixOS unstable
 - Version of the code: 0.6.0.0

### PR DESCRIPTION
**Pull request by infinisil**
Fixes #166 and ensures that such problems can't happen again by making CI build the package using `haskellPackages.buildFromCabalSdist`.

### CODE REVIEW

File: webauthn.cabal [hunk 1]
```diff
@@ -41,6 +41,7 @@ extra-source-files:
   changelog.md,
   root-certs/**/*.crt,
   tests/golden-metadata/big/blob.jwt,
+  tests/golden-metadata/big/blob-with-errors.jwt,
```
Comment (ErinvanderVeen) [inline webauthn.cabal:44]: Would it not make sense to just get everything from the golden-metadata directory?

``golden-metadata/**/*.jwt``

Comment (ErinvanderVeen) [reply webauthn.cabal:44]: That might prevent issues like this in the future?

Comment (infinisil) [reply webauthn.cabal:44]: The wildcards are unfortunately [very limited](https://cabal.readthedocs.io/en/3.4/cabal-package.html#pkg-field-extra-source-files), I wish we could do `tests/golden-metadata/*`. But yeah `golden-metadata/**/*.jwt` works, I'll change this

**Review by ErinvanderVeen — APPROVED** [review]

**Comment by infinisil** [comment]
@ErinvanderVeen Please make a new release (should be `0.6.0.1`) on Hackage

**Pull request merged**